logic.js

```diff
@@ -198,6 +198,18 @@ function load_ability_deck(deck_definition)
       }
     }
 
+    deck.new_round = function() {
+        if (this.discard.length > 0)
+        {
+            draw_ability_card(this);
+            // Maybe the last card was a reshuffle.
+            if (this.discard == 0)

```

Comment:
Agree and changing it, although it works with this.discard (javascript casts 0 == [])